internal/output/fixtures/sarif-output.md

```diff
@@ -0,0 +1,117 @@
+> for coordinating the disclosure and writing this advisory.
+> 
+> [1]: https://www.rust-lang.org/policies/security
+
+</details>
+
+
+
+## [GHSA-m5pq-gvj9-9vr8](https://osv.dev/vulnerability/GHSA-m5pq-gvj9-9vr8)
+
+<details>
+<summary>Details</summary>
+
+> > This is a cross-post of [the official security advisory][advisory]. The official advisory contains a signed version with our PGP key, as well.
+> 
+> [advisory]: https://groups.google.com/g/rustlang-security-announcements/c/NcNNL1Jq7Yw
+> 
+> The Rust Security Response WG was notified that the `regex` crate did not properly limit the complexity of the regular expressions (regex) it parses. An attacker could use this security issue to perform a denial of service, by sending a specially crafted regex to a service accepting untrusted regexes. No known vulnerability is present when parsing untrusted input with trusted regexes.
+> 
+> This issue has been assigned CVE-2022-24713. The severity of this vulnerability is "high" when the `regex` crate is used to parse untrusted regexes. Other uses of the `regex` crate are not affected by this vulnerability.
+> 
+> ## Overview
+> 
+> The `regex` crate features built-in mitigations to prevent denial of service attacks caused by untrusted regexes, or untrusted input matched by trusted regexes. Those (tunable) mitigations already provide sane defaults to prevent attacks. This guarantee is documented and it's considered part of the crate's API.
+> 
+> Unfortunately a bug was discovered in the mitigations designed to prevent untrusted regexes to take an arbitrary amount of time during parsing, and it's possible to craft regexes that bypass such mitigations. This makes it possible to perform denial of service attacks by sending specially crafted regexes to services accepting user-controlled, untrusted regexes.
+> 
+> ## Affected versions
+> 
+> All versions of the `regex` crate before or equal to 1.5.4 are affected by this issue. The fix is include starting from  `regex` 1.5.5.
+> 
+> ## Mitigations
+> 
+> We recommend everyone accepting user-controlled regexes to upgrade immediately to the latest version of the `regex` crate.
+> 
+> Unfortunately there is no fixed set of problematic regexes, as there are practically infinite regexes that could be crafted to exploit this vulnerability. Because of this, we do not recommend denying known problematic regexes.
+> 
+> ## Acknowledgements
+> 
+> We want to thank Addison Crump for responsibly disclosing this to us according to the [Rust security policy](https://www.rust-lang.org/policies/security), and for helping review the fix.
+> 
+> We also want to thank Andrew Gallant for developing the fix, and Pietro Albini for coordinating the disclosure and writing this advisory.
+
+</details>
+
+
+
+---
+
+### Affected Packages
+| Source | Package Name | Package Version |
+| --- | --- | --- |
+| lockfile:/path/to/sub-rust-project/Cargo.lock | regex | 1.5.1 |
+
+
```

Comment:
Can you add some text around "remediation instructions" around here? 

This should say something along the lines of, update the offending package past the listed fix versions, or add this to the OSV config (link to documentation) to silence this.

Reply:
Done, @hayleycd can you take a look at the `sarif-output.md` file and see if there's anything you think would be better reworded? Thanks!

Reply:
Hey @another-rex, can you give me some context on what this file is? Is this a sample of what a sarif output would look like? 

Does it change depending on what vulnerabilities are in my specific project? 

I am trying to understand the conditions under which a user would see this file. It doesn't look like we link to it in the docs at all?

Reply:
So this file is what you will see when looking at the output in `Security > Code Scanning` tab, e.g. 
![image](https://github.com/google/osv-scanner/assets/106129829/0d54eaee-bc01-4d50-ab28-7eea8fd4e9a7)

This doc is an example of what will be displayed between "Your dependency is vulnerable to..." and "Show less ^". In this case the `sarif-output.md` is just a specific output we are using to test the behavior of the code generating the SARIF file, the actual "template" is located at the top of the `sarif.go` file (https://github.com/google/osv-scanner/pull/559/files#diff-d35339a2a0d00fb1bdef55bde8021cf88c25c1817a1d1f8afcfd734a1e80a784L29)

Reply:
> Does it change depending on what vulnerabilities are in my specific project?

Yep, so the vulnerability ID, link, the contents of the Source table and Fixed table will change depending on what vulnerability this help entry is for and what lockfiles are found in the scan.

Reply:
>  Is this a sample of what a sarif output would look like?

Yep, though only a partial sample. A SARIF file can contain multiple rules (in our case ), and each rule can have a "helpText". This is what the helpText contains